src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -241,6 +243,7 @@
         "fileChanges": {
           "description": "A message relevant to this annotation.",
           "type": "array",
+          "uniqueItems": true,
```

Comment:
this is wrong. file changes needs to be a list because ordering is important, changes need to be applied in a precise sequence. also, the changes themselves could be duplicated.

'If a single fileChange object (§6.20) specifies more than one replacement, then the effect of the replacements shall be as if they were performed in the order they appear in the replacements array (§6.20.3).'